### PR TITLE
replaces all instances of "applejack" with "cider"

### PR DESCRIPTION
**Pull request by ghost**
## Changelog
:cl: Elephant Seal
fix: fixed a My Little Pony: Friendship is Magic reference in the codebase
/:cl:

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
you sure it was an mlp reference? isn't there a cider named applejack or some shit irl too?

**Comment by karmaisblackandbluepilled** [comment]
Applejack (drink)
From Wikipedia, the free encyclopedia
Applejack is a strong apple-flavored alcoholic drink produced from apples. Popular in the American colonial era, the drink's prevalence declined in the 19th and 20th centuries amid competition from other spirits.[1][2][3]

**Comment by diegoxt45** [comment]
what's even the point

**Comment by Neotwr** [comment]
actually why?

**Comment by TrickyDickRichard** [comment]
image
"horse around"

**Comment by Neotwr** [comment]
ok anti-magic friendship

**Comment by ElephantSeal** [comment]
> image
> the name of the brand of Applejack is "Buckin' Bronco" and the description has "horse around" in it

that's why I'm saying its a reference and needs to be removed

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -1950,15 +1950,15 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	description = "The perfect beverage for when you're a hick and wanna get buzzed."
 	color = "#ff6633"
 	boozepwr = 20
 	taste_description = "an honest day's work at the orchard"
-	glass_icon_state = "applejack_glass"
-	glass_name = "Applejack"
-	glass_desc = "You feel like you could drink this all neight."
+	glass_icon_state = "cider_glass"
+	glass_name = "cider"
+	glass_desc = "Drinking this makes you want to shoot degenerates violating your property."
```
Comment (yoyobatty) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:1961]: lol

**Comment by Manly-Dwarf** [comment]
obvious reference but what a weird thing to slip into the code

**Comment by GuyonBroadway** [comment]
ACKSHULLY KIDDO!!

Applejack is a real drink that's been around since the American Colonial era.

https://en.wikipedia.org/wiki/Applejack_(drink)

You're seeing pony crap in places it is not.

**Comment by TrickyDickRichard** [comment]
> ACKSHULLY KIDDO!!
> 
> Applejack is a real drink that's been around since the American Colonial era.
> 
> https://en.wikipedia.org/wiki/Applejack_(drink)
> 
> You're seeing pony crap in places it is not.

image

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 2]
```diff
@@ -396,13 +396,13 @@
 	list_reagents = list(/datum/reagent/consumable/grenadine = 100)
 	foodtype = FRUIT
 
-/obj/item/reagent_containers/food/drinks/bottle/applejack
```
Comment (ghost) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm:399]: keep applejack, only the descriptions need to go

**Comment by Neotwr** [comment]
so just change the description, retard

**Comment by GuyonBroadway** [comment]
A drink that "kicks like a horse" is an old term that I'll not let the bronies have the satisfaction of ruining.

**Comment by TrickyDickRichard** [comment]
its obviously a reference and apple cider's better anyways

**Comment by ghost** [comment]
@TrickyDickRichard it seems pretty obvious the name 'Applejack' came from the drink and not the other way around

**Comment by TrickyDickRichard** [comment]
it may not have come from it but do you think it's a coincidence they named it "Buckin' Bronco"

**Comment by ghost** [comment]
@TrickyDickRichard No, it is not. What does that change though?

**Comment by Manly-Dwarf** [comment]
"You feel like you could drink this all neight."
coincidence

**Comment by Manly-Dwarf** [comment]
the term applejack does not need to change since it's a real drink, just remove the obvious reference

**Comment by TrickyDickRichard** [comment]
rename it John Smith's All-American Applejack
"All American Applejack from the heartland, home of the brave and land of the guns."
and make it say "You feel like you could shoot some redskins."

**Pull request closed without merging**